scripts/helpers.js

```diff
@@ -0,0 +1,18 @@
+
+/**
+ * Creates a delay of the provided function, waiting ofr a delay
+ * before calling the function again.
+ * @function
+ * @param {Function} trigger - The function to delay.
+ * @returns {Function}
+ */
+export const debounce = (trigger) => {
```

Comment:
This is gold, and I'll be using it on the ~~sticky aside~~ ~~careers listing aside~~ careers listing details block work.